### DISCUSSION TITLE
Library supporting

### DISCUSSION THREAD

**Discussion by Razzwan** [vkruglikov/react-telegram-web-app#49] [Ideas]
In order to be able to maintain and develop this library, I propose to create an organization with joint management, so that more than one person has the right to merge into the main branch...

Right now, I forced to create this fork to improve it for my needs - https://github.com/altiore/twa

From my previous experience, one person will not be able support the library. Only organization with more then 2 people...

**Comment by vkruglikov** [comment]
Hi! Thank you for the discussion

This repository has a public roadmap 
https://github.com/vkruglikov/react-telegram-web-app#-roadmap

Everyone can suggest any point to discussion and feature for development if the one is suitable for the roadmap.

In the near future, I will prepare an instruction file for contributing to this repository, following which you can make any changes to the project if they coincide with the goals of the project.

**Comment by vkruglikov** [comment]
@Razzwan Thank you for the discussion! I am very appreciative of it!

I have prepared for everyone who wants to contribute to this project.
https://github.com/vkruglikov/react-telegram-web-app#contributing

Follow to steps and make changes ❤️
